pkgs/by-name/ov/oversteer/package.nix

```
@@ -87,6 +89,8 @@ stdenv.mkDerivation {
   postInstall = ''
     substituteInPlace $out/lib/udev/rules.d/* \
       --replace /bin/sh ${bash}/bin/sh
+    substituteInPlace $out/lib/udev/rules.d/* \
+      --replace /usr/bin/evdev-joystick ${linuxConsoleTools}/bin/evdev-joystick
```

```suggestion
      --replace-fail /usr/bin/evdev-joystick ${linuxConsoleTools}/bin/evdev-joystick
```

- https://github.com/NixOS/nixpkgs/issues/356002

---

Thank you for taking a look at my pr so fast. I've updated the commit message and commit.

The suggested change did not work because the function fails on the other two evdev rules. I've instead used --replace-warn. The package passes and these two messages are printed to stdout during rebuild:
```
oversteer> substituteStream() in derivation oversteer-0.8.3: WARNING: pattern /usr/bin/evdev-joystick doesn't match anything in file '/nix/store/x3rlbl61p4km8nryyzh8286a0hzfv47y-oversteer-0.8.3/lib/udev/rules.d/99-logitech-wheel-perms.rules'
oversteer> substituteStream() in derivation oversteer-0.8.3: WARNING: pattern /usr/bin/evdev-joystick doesn't match anything in file '/nix/store/x3rlbl61p4km8nryyzh8286a0hzfv47y-oversteer-0.8.3/lib/udev/rules.d/99-thrustmaster-wheel-perms.rules'
```

---

Instead of targeting `*` let's target the specific file so that we _can_ apply `--replace-fail`.